torch_ipex/csrc/cpu/dbl/Common.cpp

```diff
@@ -116,6 +122,12 @@ void equip_dil_buffer(const at::Tensor& tensor, dil::tensor dil_tensor_buffer) {
 
   IPEXTensorImpl* ipex_tensor_impl = (IPEXTensorImpl *)tensor.unsafeGetTensorImpl();
   ipex_tensor_impl->storage().set_data_ptr(std::move(shade_data_ptr));
+ 
+  // After equip_dil_buffer(), whole storage should be managed by dil tensor,
+  // and thus storage metadata should be overwritten by dil tensor 
+  // Note: Storage::set_numel() might be removed later
+  ipex_tensor_impl->storage().set_numel(dil_tensor_buffer.get_nelems());
```

Comment:
Does that mean the number of elements of dil tensor will not be as same as the number of ATen tensor (dilation/padding) ?

Reply:
We may pass a dummy tensor with zero size as output and let our ops to replace its buffer with concrete sizes.